apps/asap-server/src/entities/research-output.ts

```diff
@@ -77,6 +77,7 @@ export const parseGraphQLResearchOutput = (
     description: output.flatData?.description || '',
     tags: output.flatData?.tags || [],
     publishDate: output.flatData?.publishDate || undefined,
+    labCatalogNumber: output.flatData?.labCatalogNumber || '',
```

Comment:
why do you default to an empty string? i see the field isnt required either in the schema or in the response so i dont think this is valid. It should just be returned as undefined if missing